src/components/Simulation/ExpandableCellContent/ViewOnDhis2Btn/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+import humanize from "string-humanize";
+import { Typography, Button, makeStyles } from "@material-ui/core";
+import kebabCase from "lodash/kebabCase";
+import { useTranslation } from "react-i18next";
+import { Dhis2Icon } from "@blsq/manager-ui";
+
+import camelCase from "lodash/camelCase";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    marginBottom: theme.spacing(2),
+  },
+  icon: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
+const ViewOnDhis2Btn = props => {
+  const { cell } = props;
+  const classes = useStyles(props);
+  const { t } = useTranslation();
+
+  const isInput = !!cell.state;
+  const isOutput = !!cell.dhis2_data_element;
+  let href;
+
+  if (!isInput && !isOutput) {
+    return null;
+  }
+
+  if (isInput) {
+    href = `${
+      process.env.REACT_APP_DHIS2_URL
+    }dhis-web-maintenance/index.html#/edit/${camelCase(
+      cell.state.kind,
+    )}Section/${camelCase(cell.state.kind)}/${cell.state.ext_id}`;
+  }
```

Comment:
For me, something like this:

``` js
[process.env.REACT_APP_DHIS2_URL,
"dhis-web-maintenance",
"index.html#",
"edit",
`${camelCase(cell.state.kind)}Section`,
camelCase(cell.state.kind),
cell.state.ext_id
].join("/")
```

Is easier to read, but even then I'll always be worried if the process.env is ending with a slash or not.